modules/platforms/dotnet/Apache.Ignite.Core.Tests/Services/Model.cs

```diff
@@ -123,7 +123,7 @@ public override int GetHashCode()
     // ReSharper disable once InconsistentNaming
     public enum ACL
     {
-        Allow, Deny
+        ALLOW, DENY
```

Comment:
Why it's changed to uppercase here? AFAIK pascal case should be used for enum values in C# (https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/4x252001(v=vs.71)?redirectedfrom=MSDN). And for binary protocol enum order is used, so uppercase is not required to match dotnet and java enums.

Reply:
Actually, it's kind of one more minor .Net -> Java integration issue.
With previous enum declaration we have an exception in `ServicesTypeAutoResolveTest#TestCallPlatformServiceRemote`

```
Apache.Ignite.Core.Services.ServiceInvocationException : Proxy method invocation failed with an exception. Examine InnerException for details.
  ----> Apache.Ignite.Core.Common.IgniteException : Conflicting enum values. Name 'Allow' uses ordinal value (0) that is also used for name 'ALLOW' [typeName='org.apache.ignite.platform.model.ACL']
  ----> Apache.Ignite.Core.Common.JavaException : class org.apache.ignite.IgniteException: Conflicting enum values. Name 'Allow' uses ordinal value (0) that is also used for name 'ALLOW' [typeName='org.apache.ignite.platform.model.ACL']
```